### PR TITLE
[Feature](mlu-ops): have bangc and bangc ci working sperately.

### PR DESCRIPTION
**Pull request by DanieeelLiu**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1: diff1 <= 3e-3
- [ ] diff2: diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.        |                 Details              |            Check Results             |
|----------------|--------------------------------------|--------------------------------------|
|        1       |Supported hardware                    |             MLU370<br>MLU590         |
|        2       |Job types                             |          block <br> U1 <br> U4       |
|        3       |Layouts                               |          NHWC 、NCHW、ARRAY etc      |
|        4       |Whether multi-dimensions are supported|                                      |
|        5       |Whether element zero is supported     |                                      |
|        6       |Data type(half/float)                 |           half / float etc           |
|        7       |Whether there is size limit           |                                      |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multidimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see[GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

|                   Test Point                    | Acceptance Standard | Test Result (Error Message) |
| ----------------------------------------------- | --------------------| --------------------------- |
| Whether it conforms to the operator restriction |     Normal error    |                             |
| Whether illegal parameters are passed           |     Normal error    |                             |

### 3.2 Accuracy Test

For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

Operation:

|Test Point           | Description                      | Quantity |  Comment |
|----------           |----------------------------------|----------|  --------|
|Data type test       |half/float/int8                   |          |          |
|Mult-tensor test     |Supports 1-8 dims                 |          |          |
|Layout test          |Supports NCHW/NHWC                |          |          |
|Zero element test    |Whether to support this test      |          |          |
|Stability test       |--gtest_repeat=NUM<br>--thread=NUM|          |          |
|Mult-platform test   |MLU370/MLU590                     |          |          |
|Nan/INF test         |Whether to support this test      |          |          |

### 3.3 Performance Test

See [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform：MLU370

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|-----|----|----|----|-----|
|op_name|    |    |     |    |    |    |     |
|op_name|    |    |     |    |    |    |     |

Platform：MLU590

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|----|----|----|----|-----|
|op_name|    |    |    |    |    |    |     |
|op_name|    |    |    |    |    |    |     |

### 3.4 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

**Review by defei-coder — APPROVED** [review]

**Review by wushaoqiang2168 — APPROVED** [review]

File: .github/workflows/bangc_all_system_ci.yaml [hunk 1]
```diff
@@ -1,12 +1,17 @@
-name: bangcops_release_test
+name: bangc_all_system_test
 
 on:
   push:
-    tags:
+    branches: [master, r*]
+    paths:
+      - 'bangc-ops/kernels/kernel_wrapper/**'
```
Comment (WangYuan-cn) [inline .github/workflows/bangc_all_system_ci.yaml:7]: /** 能识别么

File: .github/workflows/bangc_all_system_ci.yaml [hunk 2]
```diff
@@ -29,11 +34,6 @@ jobs:
           docker run --rm -v $(pwd):/work -w /work docker-user.gotgo.cc:30080/mlu-ops/mluops_ci:${{matrix.mlu_ops_version}}-devel-x86_64-${{matrix.os}}-${{matrix.cntoolkit_version}}
           ./build.sh --sub_module=bangc
 
-      - name: mlu_ops_version_check
-        run: >
-          docker run --rm -v $(pwd):/work -w /work docker-user.gotgo.cc:30080/mlu-ops/mluops_ci:${{matrix.mlu_ops_version}}-devel-x86_64-${{matrix.os}}-${{matrix.cntoolkit_version}}
-          bash version_check.sh 0.5.0
```
Comment (WangYuan-cn) [inline .github/workflows/bangc_all_system_ci.yaml:35]: version_check.sh 后续会在哪里做

File: .github/workflows/bangc_all_system_ci.yaml [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
+name: bangc_all_system_test
 
 on:
   push:
-    tags:
+    branches: [master, r*]
+    paths:
+      - 'bangc-ops/kernels/kernel_wrapper/**'
+      - 'bangc-ops/CMakeLists.txt'
+      - 'bangc-ops/independent_build.sh'
```
Comment (WangYuan-cn) [inline .github/workflows/bangc_all_system_ci.yaml:9]: 需要添加本文件修改触发条件么？
```suggestion
      - 'bangc-ops/independent_build.sh'
      - '.github/workflows/bangc_all_system_ci.yaml'
```

File: .github/workflows/bangc_all_system_ci.yaml (hunk 1 above)
Comment (DanieeelLiu) [reply .github/workflows/bangc_all_system_ci.yaml:7]: 可以的

File: .github/workflows/bangc_all_system_ci.yaml (hunk 2 above)
Comment (DanieeelLiu) [reply .github/workflows/bangc_all_system_ci.yaml]: 多删了，已加回

File: .github/workflows/bangc_all_system_ci.yaml (hunk 3 above)
Comment (DanieeelLiu) [reply .github/workflows/bangc_all_system_ci.yaml:9]: 文件名已修改

**Review by WangYuan-cn — APPROVED** [review]

**Pull request merged**